website/src/blogs/how-are-tiktok-flipkart-kubesphere-and-others-using-openebs-for-local-volumes.md

```diff
@@ -29,8 +29,13 @@ Hence, Stateful Applications using local volumes must be able to tolerate this r
 
 As it happens, many of the Cloud Native Workloads - are distributed in nature and are typically deployed as StatefulSets with multiple replicas. These can sustain the failure or reduced availability of a single replica. MinIO, Redis, PostgreSQL, Kafka, Cassandra, Elastic are just some examples that are deployed using Local Volumes. For these applications - performance and consistent low latency, and ease of management are more important than the resiliency of a node to failures.
 
+<<<<<<< HEAD
```

Comment:
Looks like unresolved conflict @Pallavi-PH

Reply:
Resolved it